### PR TITLE
Updated VSAC profile to be the latest 2017 profile.

### PR DESCRIPTION
**Pull request by hossenlopp**
This has already been hotfixed on production.

### CODE REVIEW

File: lib/tasks/vsac.rake [hunk 1]
```diff
@@ -9,6 +9,7 @@ namespace :bonnie do
       effective_date = Date.parse(ENV['DATE']).strftime('%Y%m%d') if ENV['DATE']
       include_draft = !!ENV['DRAFT']
       config = APP_CONFIG["nlm"]
+      config["profile"] = ENV['PROFILE'] if ENV['PROFILE']
```
Comment (edeyoung) [inline lib/tasks/vsac.rake:12]: what does this line do?

Comment (edeyoung) [reply lib/tasks/vsac.rake:12]: nm, i get it. it grabs it from the environment variable if its specified.

**Review by edeyoung — COMMENTED** [review]
So, I checked CMS169 which was one of the measures mentioned. I was looking for the missing codes (RxNorm 1738483). I saw them in my newly uploaded measure on my account. However, when I tried the rake task, this code wasn't showing up.

Thoughts on why the rake task may not be working?

If you don't have time to look into this now, can you delete the mod to the rake task, and I'll accept this profile change?

**Comment by hossenlopp** [comment]
@edeyoung Did you use the DRAFT=true in the rake task. The profile defines the code system versions, not the value set versions.

File: lib/tasks/vsac.rake (hunk 1 above)
Comment (hossenlopp) [reply lib/tasks/vsac.rake:12]: Yeah, I wanted to be able to change the profile when using this rake task. I was tired of changing the config file and running again.

**Comment by edeyoung** [comment]
I see them now, thanks!

**Review by edeyoung — APPROVED** [review]

**Review by jbradl11 — APPROVED** [review]

**Pull request merged**